docs/tutorials/protect_against_regen.rst

```diff
@@ -0,0 +1,48 @@
+changes (for example, jumping from a 20V to a 0V command), the greater the magnitude of the regeneration current.
+
+How Can Regeneration Affect Your Module
+=========================================
+If the module's power source can absorb the regenerated current without allowing the voltage to spike, regeneration protection isn't needed. This is often true with a battery as the power source.
+When connected to a power source that cannot absorb this regenerated current, which is generally the case with benchtop power supplies, the output voltage of the source will often increase during regeneration. 
+These spikes can result in voltages above the module's and your supply's maximum rated operating voltage. When this happens, you can permanently damage the module, the power supply, or both.
+
+Protecting Against Dangerous Regeneration Voltage Spikes
+===========================================================
+Regenerative voltage spikes can pose a threat to your modules and power supply if not properly controlled. As such, all Vertiq modules can limit their regeneration current to limit the magnitude of voltage spikes on power supplies. 
+Vertiq modules have two main parameters to protect against dangerous regeneration voltage spikes. They are available through IQ Control Center's advanced tab as *Volts Limit* and *Volts Limit Starting Voltage*.
+
+For example, by default on Vertiq 81-XX Gen 2 family modules (rated to 14S), you will see:
+
+.. image:: ../_static/control_center_pics/regen_params.png
+        :alt: Regeneration Parameters in IQ Control Center
+
+In short, these two values work together in order to limit the rate at which your module will slow down or apply a braking force. In doing so, the module limits the amount of 
+negative supply current produced, reducing the voltage spike produced by the power supply.
+
+.. note::
+    These configuration parameters define **system voltages**. For example, if your module has a maximum voltage rating of 14S (or 58.8V), but your power supply has a 
+    maximum rating of 30V, the values of Volts Limit Starting Voltage and Volts Limit must be set according to the power supply's absolute 30V maximum.
+
+    As another example, suppose you want to power your Vertiq 60-08 (rated up to 12S) with a 6S power supply. In this case, you would set your voltage limits based on the
+    6S power supply.
+
+    By default, Vertiq modules have these values set to protect the module's circuits while operating with the supplied voltage at its maximum rating. In order to protect your power supply and module, 
+    please ensure that your configured voltage limits are set according to the **system's lowest maximum voltage rating**.
+
+* **Volts Limit Starting Voltage** defines the value (in volts) at which the module will begin checking if any limits on regeneration need to be applied. The value of Volts Limit Starting Voltage must always be at least 1V below Volts Limit.
```

Comment:
"Setting this value to 0V would always enable checking.  Setting this value to just below Volts Limit would still enable the maximum breaking under normal conditions, and only enable the protection once the voltage spikes."  

Or something like that.